packages/workbox-google-analytics/package.json

```diff
@@ -10,20 +10,22 @@
     "offline",
     "analytics"
   ],
-  "author": {
-    "name": "Jeff Posnick",
-    "email": "jeffy@google.com",
-    "url": "https://jeffy.info"
-  },
   "scripts": {
     "prepublish": "gulp build --project workbox-google-analytics"
   },
+  "author": "Google's Web DevRel Team",
   "license": "Apache-2.0",
   "repository": "googlechrome/workbox",
   "bugs": "https://github.com/googlechrome/workbox/issues",
   "homepage": "https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-google-analytics",
+  "dependencies": {
```

Comment:
It looks like `devDependencies` rather than `dependencies` is used elsewhere in the monorepo for cross-package dependencies. Would `devDepencencies` work for your use case?

Reply:
This should have been dependencies since day 1.

Reply:
Yeah, it should be `dependencies` or it won't get included in browser bundles.